### DISCUSSION TITLE
Updates in source audit should be reflected in mapped audit

### DISCUSSION THREAD

**Discussion by fastlorenzo** [intuitem/ciso-assistant-community#2058] [Ideas]
**Problem statement**
When I make a mapping from a project A (let's say using CCB CyFun, for example), it creates a new project B (in this example, ISO27001).
Let's assume I was only at 50% completion of the assessment of project A when I created the mapping, project B will only be partially filled.
Afterwards, I complete the assessment of project A at 100%. The changes are not reflected in project B.

**Expected behavior**
I would expect changes to be reflected in project B as this is a mapped project.

**Mock**
Install CCB CyFun, ISO27001:2022 and CCB CyFun -> ISO27001:2022 mapping.
Create a project using CCB CyFun as framework.
Fill in some data (aka assess some controls of the framework).
Assign a mapping to ISO27001:2022 framework, which will create a new project.
Fill in additional data in the first project.
Corresponding controls in the mapped project are not updated.

**Additional context**
N/A

**Comment by ab-smith** [comment]
Hello @fastlorenzo 
Thanks for the feedback. It's a good point that we've discussed during the mapping feature design as a matter of fact. 
The current implementation is a projection of the compliance status when it's triggered and we don't keep the sync because you can end up in a situation where you're updating the projection and a sync will result in a conflict and eventually information loss.
for now the easy way is to create a new projection by applying the mapping when you need a fresh sync so that it won't collide. Once we have a better approach for this UX challenge we might revisit this, somthing like a force sync button and asking the user confirmation to override and possibly lose any adjustments.

**Comment by fastlorenzo** [comment]
Hi @ab-smith, thanks for the swift pick-up of this and the insights in the decision behind this feature.
I believe the solution to propose a "sync" button might be a good compromise as it would allow the user to make a choice without needing to recreate an entire project each time :wink: 
Happy to discuss it further if you'd like!

**Comment by ab-smith** [comment]
sure, you can join the Discord server to continue our chat :)
